### PR TITLE
BC-8098 - DMED-119 - fix oeh spider

### PR DESCRIPTION
**Pull request by mamutmk5**
# Description
Fix for oeh crawler to oeh spider if the spider and the other crawlers work

## Links to Tickets or other pull requests
- https://ticketsystem.dbildungscloud.de/browse/BC-8098
- https://ticketsystem.dbildungscloud.de/browse/DMED-119

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: ansible/group_vars/edusharing.yml [hunk 1]
```diff
@@ -85,9 +85,9 @@ edusharing_crawler_list:
       memory: 512Mi
   env:
   - name: CRAWLER
-    value: oeh_importer
+    value: oeh_spider
   - name: SCHEDULE
-    value: "*-*-20:00"
+    value: "*-*-07:25"
```
Comment (aimee-889) [inline ansible/group_vars/edusharing.yml:90]: startingtime could get changed back to 20:00

File: ansible/group_vars/edusharing.yml [hunk 2]
```diff
@@ -106,6 +105,10 @@ edusharing_crawler_list:
       secretKeyRef:
         name: "{{ edusharing_instance_secret_name }}"
         key: crawleruserpassword
+  - name: PYPPETEER_WS_ENDPOINT
+    value: "ws://browserless-chromium:3000"
```
Comment (aimee-889) [inline ansible/group_vars/edusharing.yml:109]: the values could become a var

**Comment by aimee-889** [comment]
The default image should get increased: https://github.com/hpi-schul-cloud/infra-deploy/blob/main/.github/workflows/edu-crawler.yml#L18
https://github.com/hpi-schul-cloud/infra-deploy/blob/main/.github/workflows/edu-dev-crawler.yml#L14

File: ansible/group_vars/edusharing.yml (hunk 2 above)
Comment (mamutmk5) [reply ansible/group_vars/edusharing.yml]: Done

**Review by aimee-889 — APPROVED** [review]

**Pull request merged**